src/scss/templates/_collection.scss

```diff
@@ -19,3 +19,57 @@
 		}
 	}
 }
+
+.product-swatch {
+
+	&__overlay {
```

Comment:
we can probably attach these properties to a pseudo-class and save 1 DIV in DOM, please check